index.d.ts

```diff
@@ -814,6 +817,10 @@ declare namespace Eris {
   }
 
   // Message
+  interface ActionRow {
+    components: Component[];
```

Comment:
```suggestion
    components: Exclude<Component, ActionRow>[];
```

This is future proof, you could also simply use Button[] for now though. ActionRows may not include another ActionRow.

Reply:
I'll just revert it, Discord needs to do their documentation better